app/assets/stylesheets/peoplefinder/peoplefinder-ie7.scss

```diff
@@ -0,0 +1,116 @@
+@import '_colours';
+// IE7 styles
+$blackish: #6F777B;
+$darkblue: #005EA5;
+$turqoise: #93cfca  ;
+$opaqueblue: rgba(#2B8CC4, 0.25);
+$paleblue: rgba(#9BB7CE, 0.5);
+$error-text: #DF3034;
+$error-border: $error-text;
+$error-bg: #FBE1E1;
+$palegreen: #c9d8bd;
+$notice-bg: $palegreen;
+$pink-bg: #FEF7F7;
+$grey-bg: #DEE0E2;
+$green: #006435;
+$border-color: #bfc1c3;
+
+// TODO: sort out 3 misaligned fields on /people/mark-ford/edit?prompt=profile
+
+// layout stuff
+.header-app-name a{
+  color: white;
+  text-decoration: none;
+}
+.grid-wrapper{
+  clear:both;
+}
+h1, h2 {
+  margin: 30px 0;
+  border-bottom: 1px solid #6f777b;
+}
+
+
+.editing-alert {
+  background-color: $turqoise;
+  text-align: center;
+  padding:1em;
+  height: auto;
+  display: block;
+}
+.login-bar {
+  text-align: right;
+  padding: 0.5em 1em;
+  background: #DEE0E2;
+
+  span {
+    margin: 0.4em;
+  }
+}
+.breadcrumbs {
+  margin-top: 1em;
+  margin-bottom: 9px;
+  overflow: hidden;
+
+  ol {
+    li {
+      display: block;
+      float: left;
+      padding-right: 1em;
+      a{
+        text-decoration: none;
+      }
+    }
+  }
+}
+// form
+form #working-days ul,
+.working_days {
+  float: left;
+
+  li {
+    display: block;
+    float: left;
+    margin-right: 3px;
+  }
+}
+
+// profile
+#search_results {
+  .image-background{
+    float: left;
+    width:auto;
+  }
+  .details {
+    margin: 0 0 0 15px !important;
+  }
+}
+.edit-links {
+  background: $grey-3;
+  padding: 0.5em;
+  padding-bottom: 0.3em;
+  position: relative;
+  top: 15px;
+  width: 60%;
+  float: right;
+  width: 97%;
```

Comment:
I think this is the only line in `.edit-links` that we actually need in this file.